composer.json

```diff
@@ -23,9 +23,7 @@
   "require"    : {
     "php": ">=5.4.0",
     "consolidation/robo": "~1",
-    "joomla/github": "~1.3",
-    "greencape/coding-standards": "~1",
```

Comment:
I'm giving this a second thought and thinking we should go with your approach to remain with phpcs v.1 in the main branch, and using a second branch for phpcs v.2
Since you're removing the dependency here I presume it's not used at all in jorobo, though?